nsxt/resource_nsxt_policy_ipsec_vpn_service_test.go

```diff
@@ -0,0 +1,193 @@
+/* Copyright © 2022 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+var accTestPolicyIPSecVpnServiceCreateAttributes = map[string]string{
+	"display_name":  getAccTestResourceName(),
+	"description":   "terraform created",
+	"enabled":       "true",
+	"ha_sync":       "true",
+	"ike_log_level": "INFO",
+	"sources":       "192.168.10.0/24",
+	"destinations":  "192.169.10.0/24",
+	"action":        "BYPASS",
+}
+
+var accTestPolicyIPSecVpnServiceUpdateAttributes = map[string]string{
+	"display_name":  getAccTestResourceName(),
+	"description":   "terraform created",
+	"enabled":       "true",
```

Comment:
If its possible to change values of `enabled` and `action`, it would be preferrable, since we're testing update. But if it creates validation errors on platform then lets leave as is.

Reply:
`enabled` can be configured, however seems action for IpsecVpnService can only be `BYPASS`, `PROTECT` only works for IpsecVpnSession